src/Spryker/Glue/AppMerchantBackendApi/Mapper/GlueRequestMerchantAppOnboardingMapper.php

```diff
@@ -20,6 +20,7 @@ public function mapGlueRequestTransferToMerchantAppOnboardingRequestTransfer(
         $merchantTransfer->fromArray($glueRequestTransfer->getAttributes(), true);
         $merchantTransfer->setTenantIdentifier($glueRequestTransfer->getMeta()['x-tenant-identifier'][0] ?? '');
         $merchantTransfer->setMerchantReference($glueRequestTransfer->getMeta()['x-merchant-reference'][0] ?? '');
+        $merchantTransfer->setName($glueRequestTransfer->getMeta()['x-merchant-name'][0] ?? '');
```

Comment:
How this `mapGlueRequestTransferToMerchantAppOnboardingRequestTransfer` method is related to payment transfer?